src/shared/js/utils/functions.js

```diff
@@ -441,6 +441,36 @@ const tablerSvg = (pathName, id, classNames) => {
+                <path d="M 22.597826 37.345109 C 23.673913 37.345109 24.54212 38.213315 24.54212 39.289402 L 24.54212 40.316576 C 24.54212 41.392663 23.673913 42.273098 22.597826 42.273098 C 21.521739 42.273098 20.641304 41.392663 20.641304 40.316576 L 20.641304 39.289402 C 20.641304 38.213315 21.521739 37.345109 22.597826 37.345109 Z M 22.597826 37.345109 " transform="matrix(0.319444,0,0,0.319444,0,0)"/>
+                <path d="M 27.758152 39.228261 C 28.834239 39.228261 29.702446 40.096467 29.702446 41.172554 L 29.702446 42.199728 C 29.702446 43.275815 28.834239 44.15625 27.758152 44.15625 C 26.682065 44.15625 25.80163 43.275815 25.80163 42.199728 L 25.80163 41.172554 C 25.80163 40.096467 26.682065 39.228261 27.758152 39.228261 Z M 27.758152 39.228261 " transform="matrix(0.319444,0,0,0.319444,0,0)"/>
+                <path d="M 33.052989 46.797554 C 34.129076 46.797554 34.997283 47.665761 34.997283 48.741848 L 34.997283 49.78125 C 34.997283 50.857337 34.129076 51.725543 33.052989 51.725543 C 31.976902 51.725543 31.108696 50.857337 31.108696 49.78125 L 31.108696 48.741848 C 31.108696 47.665761 31.976902 46.797554 33.052989 46.797554 Z M 33.052989 46.797554 " transform="matrix(0.319444,0,0,0.319444,0,0)"/>
+                <path d="M 20.188859 56.53125 L 19.980978 43.067935 C 19.907609 41.649457 18.745924 40.524457 17.327446 40.5 C 16.630435 40.475543 15.970109 40.744565 15.480978 41.221467 C 15.004076 41.710598 14.735054 42.383152 14.759511 43.067935 L 14.808424 59.686141 " transform="matrix(0.319444,0,0,0.319444,0,0)"/>
+                <path d="M 30.58288 56.800272 L 30.411685 41.625 C 30.411685 41.625 30.154891 39.044837 27.868207 39.044837 C 25.569293 39.044837 25.165761 41.625 25.165761 41.625 " transform="matrix(0.319444,0,0,0.319444,0,0)"/>
+                <path d="M 35.767663 59.686141 L 35.633152 48.754076 C 35.461957 47.519022 34.373641 46.601902 33.126359 46.638587 C 31.842391 46.589674 30.717391 47.482337 30.497283 48.754076 " transform="matrix(0.319444,0,0,0.319444,0,0)"/>
+                <path d="M 25.251359 55.235054 L 25.116848 39.142663 C 24.945652 37.907609 23.857337 37.002717 22.610054 37.039402 C 21.326087 36.978261 20.201087 37.883152 19.980978 39.154891 L 20.005435 44.755435 " transform="matrix(0.319444,0,0,0.319444,0,0)"/>
+                <path d="M 38.995924 36.5625 L 45.415761 26.902174 " transform="matrix(0.319444,0,0,0.319444,0,0)"/>
+                <path d="M 36.085598 43.532609 C 36.085598 43.532609 37.956522 42.138587 38.519022 41.612772 C 39.558424 40.78125 40.120924 39.497283 39.998641 38.164402 C 39.839674 37.173913 38.922554 36.501359 37.932065 36.648098 " transform="matrix(0.319444,0,0,0.319444,0,0)"/>
+                <path d="M 43.153533 25.410326 L 52.813859 11.017663 L 58.035326 13.769022 L 48.203804 28.748641 L 45.415761 26.902174 Z M 43.153533 25.410326 " transform="matrix(0.319444,0,0,0.319444,0,0)"/>
+            </svg>`;
+
+        case "ar5iv":
```

Comment:
It's pretty ugly, maybe remove it and just use "message"

Reply:
yeah that one's too much.

I think `square-rounded-number-5` from https://tabler-icons.io/ would be good enough